vtr_flow/arch/multi_die/aman_3d_coffe.xml

```diff
@@ -0,0 +1,1597 @@
+<architecture>
```

Comment:
Add comment to top of architecture. Where did it come from, what's it for, any caveats?
(Can add a reference to the fpt paper if it has info on this architecture).
Rename to something other than aman (describe what it is, not who wrote it).
Does his have delays between dice?  If not, update it to have some reasonable switch delay between dice.

Reply:
I have to fix a bunch of 3D architectures and add to them. Also, in another branch, I am working on developing some more tests on 3D architecture too. These comments are added on that branch.